### DISCUSSION TITLE
Feature Request - Manually sort tiles

### DISCUSSION THREAD

**Discussion by SteveSinclair41** [AntonyCorbett/OnlyM#412] [Ideas]
Apologies brother if this is a setting that I have overlooked. I would like to be able to order the media in the list. The only way I can see to do this is to rename the files and number them with a prefix 1,2,3,4,5 ...
Would you be able to add a feature to move the media up/down in the panel?

**Comment by AntonyCorbett** [comment]
@SteveSinclair41 Thanks for your suggestion. This has been discussed a few times and you will find the commentary in the Issues page. In particular please see my comment here: https://github.com/AntonyCorbett/OnlyM/issues/386#issuecomment-850899760

It explains some of the difficulties.
